x/programs/runtime/runtime.go

```diff
@@ -44,6 +44,27 @@ type runtime struct {
 	log    logging.Logger
 }
 
+func (r *runtime) Create(ctx context.Context, programBytes []byte) (uint64, error) {
+	err := r.Initialize(ctx, programBytes)
+	if err != nil {
+		return 0, err
+	}
+	// get programId
+	programID := initProgramStorage()
+
+	// call initialize
+	result, err := r.Call(ctx, "init", uint64(programID))
```

Comment:
init in this context is perhaps required by the examples as they are now but as stated above the programID is going to be the result something like a `create_program` action. I would like us moving towards that direction in each iteration if possible.